sgkit/stats/aggregation.py

```diff
@@ -167,6 +200,53 @@ def count_variant_alleles(ds: Dataset, merge: bool = True) -> Dataset:
     return conditional_merge_datasets(ds, new_ds, merge)
 
 
+def count_cohort_alleles(ds: Dataset, merge: bool = True) -> Dataset:
+    """Compute per cohort allele counts from genotype calls.
+
+    Parameters
+    ----------
+    ds
+        Genotype call dataset such as from
+        `sgkit.create_genotype_call_dataset`.
+    merge
+        (optional)
+        If True (the default), merge the input dataset and the computed
+        output variables into a single dataset, otherwise return only
+        the computed output variables.
+        See :ref:`dataset_merge` for more details.
+
+    Returns
+    -------
+    Dataset containing variable `call_allele_count` of allele counts with
+    shape (variants, cohorts, alleles) and values corresponding to
+    the number of non-missing occurrences of each allele.
+    """
+
+    n_variants = ds.dims["variants"]
+    n_alleles = ds.dims["alleles"]
+
+    ds = count_call_alleles(ds)
+    AC, SC = da.asarray(ds.call_allele_count), da.asarray(ds.sample_cohort)
```

Comment:
Should we standardize on using `ds. call_allele_count` vs `ds['call_allele_count']`?  I've been trying to use the former in examples only and stick to the latter in code, but not for great reasons other than consistency and hedging against potentially having variable names that aren't legal attributes or conflict with something in the xarray namespace.  Presumably not using attributes becomes easier to standardize on with https://github.com/pystatgen/sgkit/pull/276.

Reply:
I'm not sure. It's certainly shorter to say `ds.call_allele_count`, but this form can't be used in assignments:

```python
ds.call_allele_count = ...
```

I've changed the accessors in this PR, but we should probably discuss a bit more about our code standards, and what we recommend for user code.